src/vm/runners/cairo_runner.rs

```diff
@@ -277,6 +277,10 @@ impl<'a> CairoRunner<'a> {
         Ok(hint_data_dictionary)
     }
 
+    pub fn get_constants(&self) -> &HashMap<String, BigInt> {
+        &self.program.constants
+    }
```

Comment:
@fmoletta please add a test for this method

Reply:
Test added!